lib/src/base-styles/_nx-lists.scss

```diff
@@ -63,6 +63,7 @@
     @include regular();
 
     border: none;
+    line-height: 1.5;
```

Comment:
If this is specifically meant to be in sync with the main line-height should it use a variable?

Reply:
> If this is specifically meant to be in sync with the main line-height should it use a variable?

Sure: https://github.com/sonatype/sonatype-react-shared-components/pull/24/commits/b2e9b5427ab6596367636ec80aee3a05d85f9c0e